app/src/main/res/layout/activity_login.xml

```diff
@@ -1,16 +1,151 @@
 <?xml version="1.0" encoding="utf-8"?>
-<layout>
+<layout xmlns:android="http://schemas.android.com/apk/res/android">
 
     <data>
 
     </data>
 
-    <android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
-        xmlns:app="http://schemas.android.com/apk/res-auto"
-        xmlns:tools="http://schemas.android.com/tools"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        tools:context=".ui.login.LoginActivity">
 
-    </android.support.constraint.ConstraintLayout>
+        <ScrollView
```

Comment:
Why do we need scroll view ? it's a single page view.If u wanna push the screen up on keyboard opening use adjust pan in manifest file. 
https://stackoverflow.com/questions/17410499/difference-between-adjustresize-and-adjustpan-in-android

Reply:
Scroll view is used if phone is rotated

Reply:
Develop the application only in portrait mode

Reply:
Changed the LoginActivity to portrait mode